AUTHORS.md

```diff
@@ -39,7 +39,8 @@ We do not forget all translators, for their work of translating Element into man
 
 Feel free to add your name below, when you contribute to the project!
 
-Name      | Matrix ID                   | GitHub
-----------|-----------------------------|--------------------------------------
-gjpower   | @gjpower:matrix.org         | [gjpower](https://github.com/gjpower)
-TR_SLimey | @tr_slimey:an-atom-in.space | [TR-SLimey](https://github.com/TR-SLimey)
+Name         | Matrix ID                   | GitHub
+-------------|-----------------------------|--------------------------------------
+gjpower      | @gjpower:matrix.org         | [gjpower](https://github.com/gjpower)
+TR_SLimey    | @tr_slimey:an-atom-in.space | [TR-SLimey](https://github.com/TR-SLimey)
+CicadaCinema |                             | [CicadaCinema](https://github.com/CicadaCinema)
```

Comment:
This file is completely upstream and has not been Schildi-fied by any means. Accordingly, it has no meaning for SchildiChat, and if you change it, it will just cause avoidable merge conflicts. Would it be fine to you to add your name to `FEATURES.md` instead (e.g., by adding something like "contributed by")? Alternatively, I'd need to Schildi-fy AUTHORS.md and include it into my merge scripts, which would also be an option, but I didn't really see the need for that so far.